src/components/TableMatrix/TableHeader/TableHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from '../TableHeader/TableHeader.module.scss';
+
+const topCells = (value: string | number) => {
```

Comment:
in this case, it is better to pass arguments in string type
you can pass arguments like _**topCells(String(i + 1))**_